cordova-plugin-appcenter-shared/src/android/AppCenterSharedPlugin.java

```diff
@@ -22,6 +22,21 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
             AppCenterUtils.sendUUIDPluginResultFromFuture(AppCenter.getInstallId(), callbackContext);
             return true;
         }
+        if (action.equals("setUserId")) {
```

Comment:
add a newline after the closing brace (seems to be the style in the rest of the repo)

Reply:
Fixed, thanks!